src/v2/templates/people/person-list.php

```diff
@@ -16,6 +16,18 @@
 use ChurchCRM\model\ChurchCRM\PersonCustomMasterQuery;
 use ChurchCRM\model\ChurchCRM\PropertyQuery;
 
+
+function emptyOrUnassigned($stuff){
```

Comment:
nit: space in between bracing

```suggestion
/**
 * This will avoid to call the db twice one to check if empty the other one to return the value
 * no caching was being done by the ORM so lets keep the value and return if not empty
 *
 * @var mixed $stuff
 */
function emptyOrUnassigned($stuff) {
```